### PR TITLE
preview list of articles in grid

### PR DESCRIPTION
**Pull request by vinyll**
Screen Shot 2019-12-26 at 16 45 07

### CODE REVIEW

File: content/components.html [hunk 1]
```diff
@@ -356,66 +356,59 @@ <h2>Neomad profiles</h2>
...
     <div class="st-container st-full">
-      <article class=preview>
-        <div class=bar>&nbsp;</div>
-        <div class=bar>&nbsp;</div>
-        <div class=bar>&nbsp;</div>
-        <div class=bar>&nbsp;</div>
-        <a href>Write this article</a>
-      </article>
-    </div>
+      <section class=previews-list>
```
Comment (fuentesloic) [inline content/components.html]: I'm still not good with the class naming but WDYT about `previews` or `preview-list`?

**Review by fuentesloic — COMMENTED** [review]
Open question, if you define this grid with 3 articles in row (wrapping if more I guess?), it's because you want the same display on neomad profile article list?

File: content/components.html (hunk 1 above)
Comment (vinyll) [reply content/components.html]: I saw we had used a class `.article-list`, hence the name

Comment (fuentesloic) [reply content/components.html]: yes but `article-list` not `articles-list`, WDYT?

Comment (vinyll) [reply content/components.html]: It actually doesn't matter to me. Which do you prefer?

Comment (fuentesloic) [reply content/components.html]: `.preview-list` just to keep consistance ^^

**Review by fuentesloic — APPROVED** [review]

**Pull request merged**